recipe-server/client/actions/opt-out-study/package.json

```diff
@@ -30,6 +30,11 @@
                     "type": "string",
                     "format": "uri",
                     "minLength": 1
+                },
+                "isEnrollmentPaused": {
```

Comment:
I was going to suggest that this setting needs to be documented, but I don't see any docs at all for this action [on readthedocs](https://normandy.readthedocs.io/en/latest/user/actions/index.html). Should we add those docs in a separate PR?